src/renderer/components/RuntimeAdapter.tsx

```diff
@@ -42,15 +45,17 @@ export default function RuntimeAdapter({
       async ({ accountId: notificationAccountId, msgId, chatId }) => {
         if (accountId !== notificationAccountId) {
           closeAllDialogs()
+          // selectAccount always loads the last used chat
+          // by setting saveLastChatId we force to load immediately
+          // the chat holding the notification message
+          await saveLastChatId(notificationAccountId, chatId)
           await window.__selectAccount(notificationAccountId)
-        }
-
-        if (chatId !== 0) {
+        } else if (chatId !== 0) {
+          await selectChat(accountId, chatId)
```

Comment:
jumpToMessage also selects chat, so not sure if this is needed here.

Reply:
Ok thanks for paying attention. looks like I confused this with select chat, sorry for being wrong.